README.md

```diff
@@ -32,6 +33,20 @@ Documentation is available at [geopandas.org](http://geopandas.org)
 [Read the Docs](http://geopandas.readthedocs.io/en/latest/)
 (release and development versions).
 
+The GeoPandas project uses an [open governance model](https://github.com/geopandas/governance/blob/main/Governance.md)
```

Comment:
```suggestion
[//]: # (numfocus-fiscal-sponsor-attribution)

The GeoPandas project uses an [open governance model](https://github.com/geopandas/governance/blob/main/Governance.md)
```

Just missing the markdown comment which is required for automated checks (https://github.com/numfocus/templates/blob/master/fiscal-sponsor-readme-attribution.md?plain=1)